zitadel/terraform/applications.tf

```diff
@@ -0,0 +1,45 @@
+resource zitadel_application_api application_api {
```

Comment:
just another point came to my mind. When we create applications, there is client id always generated and in case of application_api, even a secret key is generated. 
Can you please see if this can be clearly written in the output of the script as a log. So that the user can later use it to configure the consumer of these applications. 

If this works, we must add this point too in the readme file.

Reply:
fixed